dotcom-rendering/src/lib/getAdPositions.ts

```diff
@@ -74,6 +75,34 @@ export const getMobileAdPositions = (
 		// Should insert no more than 10 ads
 		.slice(0, 10);
 
+/**
+ * Uses a very similar approach to pressed fronts, except we
+ * - Don't need to consider thrashers
+ * - Don't need to consider the 'most viewed' container
```

Comment:
Slightly more readable for the negative?

```suggestion
 * - Do not need to consider thrashers
 * - Do not need to consider the 'most viewed' container
```